src/_posts/databases/mysql/2000-01-01-monitoring.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Monitoring and Auditing Your Scalingo for MySQL® Addon
+nav: Monitoring and Auditing
+modified_at: 2024-03-05 12:00:00
+tags: databases mysql addon
+index: 7
```

Comment:
question: is it expected there is no index 5 or 6?

Reply:
Actually the Managing page (https://github.com/Scalingo/documentation/pull/2501/files#diff-685fc38e067ae46ecbf376de0ddb5b4ad26de94a8334dde492c21345f5a0c7b0) has index 5.
I can't tell why there's no index 6. I probably scr*wd up :sweat_smile:

Anyway, it's fixed in https://github.com/Scalingo/documentation/pull/2501/commits/703061e3054bdda288419ff93634d5b1407f0337